addons/resource.language.en_gb/resources/strings.po

```diff
@@ -21955,3 +21955,15 @@ msgstr ""
 msgctxt "#39118"
 msgid "Share debug log"
 msgstr ""
+
+#. Label for enabling scaling filter for better Pixel Art Games
+#:xbmc/settings/settings.xml
+msgctxt "#39119"
+msgid "Enable Integer Scaling Filter (NN)"
+msgstr ""
+
+#. Description of setting with label #39119 "Enable Integer Scaling Filter (NN)"
+#: system/settings/settings.xml
+msgctxt "#39120"
+msgid "Integer scaling (IS) is a nearest-neighbor upscaling technique that simply scales up the existing pixels by an integer (i.e., whole number) multiplier.Nearest-neighbor (NN) interpolation works by filling in the missing color values in the upscaled image with that of the coordinate-mapped nearest source pixel value"
```

Comment:
I would propose a slight simplification (and space between sentences) of the help text.

Before:

> Integer scaling (IS) is a nearest-neighbor upscaling technique that simply scales up the existing pixels by an integer (i.e., whole number) multiplier.Nearest-neighbor (NN) interpolation works by filling in the missing color values in the upscaled image with that of the coordinate-mapped nearest source pixel value"

After:

> Integer scaling (IS) is an upscaling technique that scales up the existing pixels by a whole number. Nearest-neighbor (NN) interpolation works by filling in the missing color values with that of the nearest source pixel value.

Reply:
I think the emphasis here is that 'IS' is a special case of the nearest-neighbor (NN) scaling technique. The nearest-neighbor technique works by generating the missing pixels information from neighboring pixels. Now, in the case of IS, as the target resolution is Integer times that of the source resolution, the whole pixel can be repeated, giving better clarity in the output.

Reply:
You're right, better would be:

> Integer scaling (IS) is a nearest-neighbor upscaling technique

While I oversimplified, I think when it comes to reading text on a TV, there's more value in less.

Reply:
Agree